packages/website/components/layout.js

```diff
@@ -32,7 +34,7 @@ export default function Layout({
   const shouldWaitForUser = needsUser && status === 'loading'
 
   return (
-    <div>
+    <div className={clsx(pageBgColor, 'min-h-screen')}>
```

Comment:
We need to add `flex flex-col` here to solve the layout issue

Reply:
Was gonna leave that for another task, but I added now